.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#package-ecosystem-
```

Comment:
suggestion to move the dependabot changes into a separate PR.

Reply:
Reverted